content/add-ons/rivet-datepicker.md

```diff
@@ -0,0 +1,235 @@
+While it's possible to overwrite these defaults when you create a new datepicker instance, doing so will create UX and accessibility issues. **Please do not overwrite these default configurations.**
+{{< /alert >}}
+
+## Advanced options
+You can pass any of the other [available Pikaday options](https://github.com/Pikaday/Pikaday#configuration) to the `RivetDatepicker` instance.
+
+{{< code lang="js" >}}
+const startDate = new RivetDatepicker({
+  field: document.getElementById('start-date'),
+  disableWeekends: true
+});
+{{< /code >}}
+
+In addition, all [Pikaday methods](https://github.com/Pikaday/Pikaday#methods) are available on the `RivetDatepicker.picker` property:
+
+{{< code lang="js" >}}
+const startDate = new RivetDatepicker(options);
+
+// Destroys the "startDate" instance and removes all event listeners
+startDate.picker.destroy();
+{{< /code >}}
+
+## Date formatting options
+Pikaday has a number of options you can use to format dates. The Pikaday `toString` option can be passed a function that accepts the `Date` object and returns a String in the format you'd like. In the examples on this page, we are using a function called `formatRivetDate`:
+
+{{< code lang="js" >}}
+function formatRivetDate(date) {
+  const day = date.getDate();
+  const month = date.getMonth() + 1;
+  const year = date.getFullYear();
+  return month + '-' + day + '-' + year;
+}
+{{< /code >}}
+
+You can then pass your custom date format function to the `toString` options:
+
+{{< code lang="js" >}}
+const startDate = new RivetDatepicker({
+  field: document.getElementById('start-date'),
+  toString: function(date, format) {
+    return formatRivetDate(date);
+  }
+});
+{{< /code >}}
+
+### Using Moment.js
+The Rivet datepicker add-on supports the use of the date formatting library [Moment.js](http://momentjs.com/). See more about [using Moment to format dates](https://github.com/Pikaday/Pikaday#formatting) in the [Pikaday documentation](https://github.com/Pikaday/Pikaday#formatting).
+
+{{< alert variant="warning" title="Moment.js file size" >}}
+While Moment.js is a powerful and useful library, it is quite large and can add significant overhead to your application. If you are only using Moment.js to format dates with the Rivet datepicker, you might consider writing your own formatting function instead rather than importing the entire Moment.js library.
+{{< /alert >}}
+
+## Date and time input
+The Rivet calendar datepicker does not support a time picker option.
+
+If you are asking users to input date and time, we recommend using separate inputs to capture the time value and concatenating the date and time on the client or server.
+
+The example below uses an `<input type="time">` element. In [browsers that support time inputs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/time#Handling_browser_support), users will see a field specially-formatted for entering a time.
+
+### 2. Add the markup to your HTML
```

Comment:
Looks like there is some stuff leftover from the switch docs in here. It should be removed.